pkg/privilege/privileges/cache.go

```diff
@@ -625,19 +627,27 @@ func compareColumnsPrivRecord(x, y columnsPrivRecord) int {
 func compareHost(x, y string) int {
```

Comment:
Why do you change it?

Reply:
See this change of test https://github.com/pingcap/tidb/pull/61677/files#diff-860eeb97f0492c2eeb38ecca6442094a483cb30f9afe7aac9f6ec4ce6b0d9a6bR413-R414